### PR TITLE
dedge/api entityid

### PR DESCRIPTION
**Pull request by dedgems**
- Adds base for updating Microsoft.KeyVault from version preview/7.3-preview to version 7.4-preview
- Updates readme
- Updates API version in new specs and examples
- Key Vault 7.4-preview to 7.4-preview.1 (#15900)
- Key Vault keys API: Add OKP key type and EdDSA digital signature (#15989)
- Add EntityId to the API definitions for secrets, keys, and certificates

### CODE REVIEW

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/certificates.json [hunk 1]
```diff
@@ -1331,6 +1331,11 @@
           "type": "string",
           "format": "base64url",
           "description": "Thumbprint of the certificate."
+        },
+        "entityId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The system-generated ID for this secret."
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/certificates.json:1338]: Seems to be a copy/paste error: you mention "secret" here for certificates. Same for keys. Please be sure to update them all.

**Review by heaths — CHANGES_REQUESTED** [review]
The example responses (see under "examples" directory) should probably be updated as well. Not sure how @jlichwa or others usually do it, but might just be enough to hand-code some sample values rather than record them as long as the model structure is correct.

**Review by heaths — APPROVED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by heaths** [comment]
@lmazuel the [check](https://github.com/Azure/azure-rest-api-specs/pull/17854/checks?check_run_id=5240084358) completed successfully but didn't report back status, even after re-running it. Could you use your magic powers, please?

**Pull request merged**